slave/process-o365-contacts-export/changelog

```diff
@@ -0,0 +1,5 @@
+perun-slave-process-o365-contacts-export (3.0.0) stable; urgency=low
+
+  * Package for exporting o365 conctacts from Perun to o365
```

Comment:
Please replace with following:
"Package for exporting Perun managed o365 emails"